lib/aur-fetch

```diff
@@ -150,10 +140,16 @@ if ! (($#)); then
     usage
 fi
 
-if [[ -v log_dir ]] && [[ ! -d $log_dir ]]; then
-    printf >&2 '%s: %q: not a directory' "$argv0" "$log_dir"
-    exit 21
-fi
+case $log in
+    directory)
+	# store patches in current directory by default
+        if [[ ! -d ${patch_dir:=$startdir} ]]; then
+            error "$argv0: '$patch_dir': not a directory"
```

Comment:
`error()` is used but `source /usr/share/makepkg/util/message.sh` is missing.

output is monochromatic due to lack of 
```
if [[ -t 2 && ! -o xtrace ]]; then
    colorize
fi
```

Message is missing a trailing new line. 

```suggestion
            error '%s: %s\n: not a directory' "$argv0" "$patch_dir"
```

Reply:
`error` doesn't take formatters - it roughly corresponds to `printf %s\n' - but the output should be colorized in any case.

Reply:
It's personal preference, I don't mind either style, but error does work with formatters. E.g. https://git.archlinux.org/pacman.git/tree/scripts/makepkg.sh.in#n246 and in the source from error itself you can see that "$@" is passed through.